numba_cuda/numba/cuda/cudadrv/nvrtc.py

```diff
@@ -233,12 +232,16 @@ def compile(src, name, cc):
     #   being optimized away.
     major, minor = cc
     arch = f'--gpu-architecture=compute_{major}{minor}'
-    include = f'-I{config.CUDA_INCLUDE_PATH}'
+
+    cuda_include = f"-I{get_cuda_paths()['include_dir'].info}"
```

Comment:
Following the suggestion above, it sounds like we can simply augment the existing searching of `config.CUDA_INCLUDE_PATH` by making this

```suggestion

    cuda_include = [f"-I{get_cuda_paths()['include_dir'].info}",
                    f"-I{config.CUDA_INCLUDE_PATH}"]
```